### DISCUSSION TITLE
Filter by columns in Manage Data

### DISCUSSION THREAD

**Discussion by GourmetSaint** [mealie-recipes/mealie#3401] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

There is no way to, for example, to filter all Recipes by a tag or category and add or remove tags or categories. Data migrated from the apps often have a plethora of different tags applied as part of the migration process.

### Please provide a concise description of the feature that would resolve your issue.

In the 'Manage Data' section, enable filtering by column to display a selected subset of data. E.g. in Recipes, filter in Tags column to select all recipes with a particular Tag or Tags. User can then select all and perform Action of adding Category or another Tag. Very useful for manipulating data after Migration from other Recipe Management apps.

### Please consider and list out some caveats or tradeoffs made in your design decision

Adds to existing functionality.  Common method for data management with intuitive hmi as reminiscent of spreadsheet editing.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by joaorgoncalves** [comment]
Seems similar to https://github.com/mealie-recipes/mealie/discussions/2734.
